sefaria/helper/text.py

```diff
@@ -1,6 +1,10 @@
 # encoding=utf-8
+import unicodecsv as csv
```

Comment:
In Python 3, `import csv` is enough. `csv` handles unicode.

Reply:
but the lines:
    output = io.BytesIO()
    report = csv.writer(output)
won't work in the new csv module, and they were used here 3 times